packages/catalog-process/src/model/domain/apiConverter.ts

```diff
@@ -141,7 +141,6 @@ export const documentToApiDocument = (
   contentType: document.contentType,
   prettyName: document.prettyName,
   path: document.path,
-  uploadDate: document.uploadDate,
```

Comment:
Hi @taglioni-r in this [PR](https://github.com/pagopa/interop-be-monorepo/pull/265/files#diff-a912256c8205d063d7abbec28c7463460a613e22b78ee926fa6e935630d4aebc) you added this `uploadDate`, but it seems it is not present in the model, could you please check?

Reply:
Thanks! I didn't notice it.